src/BALLFALL.C

```diff
@@ -515,8 +509,6 @@ void intialiseObject(Object* objects, int count)
     }
 }
 
-void drawObject(Object*);
```

Comment:
Error: drawObject called (in moveObject) before it declaration.
i think this is the last error you going to face in this code. 😄

Reply:
We should be able to just move the `drawObject` definition before the `moveObject` definition to resolve this as I do not see any functions that the drawObject function itself calls. I'm making this adjustment now and will commit shortly.